README.md

```diff
@@ -100,13 +100,33 @@ on [vuestic.dev](https://vuestic.dev)
 
 ### Partners & Sponsors ❤️
 <div>
-        <a href="https://nuxt.com/" target="_blank" title="Nuxt - The Intuitive Web Framework"><img hspace="15" vspace="10" src="./.github/assets/nuxt.svg" loading="lazy" alt=""></a>
-        <a href="https://epicmax.co/" target="_blank" title="Epicmax - Top Vue.js Development Company"><img src="./.github/assets/epicmax.svg" loading="lazy" vspace="10" alt=""></a>
-        <a href="https://www.ag-grid.com/" target="_blank" title=" Data Grid: AG Grid: High-Performance React Grid, Angular Grid, JavaScript Grid"><img hspace="15" vspace="10" src="./.github/assets/aggrid.svg" loading="lazy" alt=""></a>
-        <a href="https://vuejobs.com/" target="_blank" title="Vue.js jobs – Browse through dozens of Vue.js openings"><img src="./.github/assets/vuejobs.svg" loading="lazy" vspace="10" alt=""></a>
-        <a href="https://flatlogic.com/" target="_blank" title="Flatlogic - The best way to create React, Angular and Vue full-stack web applications"><img hspace="15" vspace="10" src="./.github/assets/flatlogic.svg" loading="lazy" alt=""></a>
+  <a href="https://nuxt.com/" target="_blank" title="Nuxt - The Intuitive Web Framework">
```

Comment:
Looks like it can be rewritten using `v-for` (in purpose not to make changes to the each item in a separate way), but up to you.

Reply:
This is README.md file. :)

Reply:
Ah, it's md, sorry xD